examples/nlp/Tweet-classification-using-TFDF.py

```diff
@@ -0,0 +1,215 @@
+"""
+Title: Text Classification using TFDF and Pre-trained embeddings
+Author: Gitesh Chawda
+Date created: 15/04/2022
+Last modified: 15/04/2022
+Description: Using Tensorflow Decision Forest for text classification
+"""
+"""
+## Introduction
+
+[TensorFlow Decision Forests](https://www.tensorflow.org/decision_forests) (TF-DF)
+provide powerful models, especially with structured data,
+It is a collection of state-of-the-art algorithms of Decision Forest models that are
+compatible with Keras APIs. The module includes Random Forests, Gradient Boosted Trees,
+and CART, and can be used for regression, classification, and ranking tasks.
+
+Alternatively for getting started
```

Comment:
Can you add a link to the "Using text and neural network features" official tutorial:
https://www.tensorflow.org/decision_forests/tutorials/intermediate_colab

TF-DF can consume text feature natively (see https://arxiv.org/abs/2009.09991). This is slower than using pre-trained embedding, but it is often simpler to use, give good results, and combines well with pre-trained embedding (when one is available). It is worth mentioning it in the introduction.

Reply:
Yeah sure I will add a reference link